src/python3_pip_skeleton/__main__.py

```diff
@@ -0,0 +1,16 @@
+from argparse import ArgumentParser
```

Comment:
Not using typer?

Reply:
No, that would add a non-stdlib runtime dependency which I think we want to avoid